src/number_theory/diophantine_approximation.lean

```diff
@@ -6,16 +6,26 @@ Authors: Michael Geißer, Michael Stoll
 import tactic.basic
 import data.real.irrational
 import combinatorics.pigeonhole
+import ring_theory.coprime.lemmas
+import data.int.units
+import algebra.continued_fractions.computation.translations
+import algebra.continued_fractions.computation.approximation_corollaries
 
 /-!
 # Diophantine Approximation
 
-This file gives proofs of various versions of **Dirichlet's approximation theorem**
-and its important consequence that when `ξ` is an irrational real number, then there are
-infinitely many rationals `x/y` (in lowest terms) such that `|ξ - x/y| < 1/y^2`.
-
+The first part of this file gives proofs of various versions of
+**Dirichlet's approximation theorem** and its important consequence that when $\xi$ is an
+irrational real number, then there are infinitely many rationals $x/y$ (in lowest terms)
+such that
+$$\left|\xi - \frac{x}{y}\right| < \frac{1}{y^2} \,.$$
```

Comment:
Is this actually an improvement to the previous `|ξ - x/y| < 1/y^2`?

Reply:
It should look better in the HTML docs, but I'm not sure what the conventions are (or whether there are any conventions regarding the use of LaTeX in docstrings).